### PR TITLE
[Docs] Minor docs polish

### PR DESCRIPTION
**Pull request by codeman7**
While reviewing the release I noticed some extra white spaces within the docs and an extra character in documentation. This PR cleans that up.

### CODE REVIEW

**Comment by romoore** [comment]
LGTM. We'll have to wait until changes are allowed into the sprint to merge this.

File: components/Ripple/docs/typical-use.md
Comment (jverkoey) [inline components/Ripple/docs/typical-use.md:266]: Please re-run the docs generator for this component.

**Review by jverkoey — APPROVED** [review]

File: components/Ripple/docs/typical-use.md
Comment (codeman7) [reply components/Ripple/docs/typical-use.md:266]: Done

**Pull request merged**
